### ISSUE TITLE
Error in Step4 Predict Events

### ISSUE THREAD

**Issue by Chongsyl** [tianchengzhe/EllipTrack#2]
OS: Windows
MATLAB: 2020a

I am running the mainfile after segmentation and creating a light training set (possibly 40 marked cells) and re-specified the parameters for segmentation with the loaded training set. I reran the entire mainfile.m and come across this error:

"Unable to perform assignment because the left and right sides have a different number of elements.

Error in compute_migration_sigma (line 67)
                    all_migration_sigma{i}(j) = sigma_val(density_val==num_cells);

Error in compute_score_migration (line 22)
all_migration_sigma = compute_migration_sigma( prob_para, all_training_data, all_num_ellipses,
all_ellipse_info, size_image, frames_to_track );

Error in step4_predict_events (line 50)
        [ all_prob_migration{row_id, col_id, site_id}, all_prob_inout_frame{row_id, col_id, site_id},
        all_motion_classifiers{row_id, col_id, site_id}, all_migration_sigma{row_id, col_id, site_id} ] =
        compute_score_migration ( ...

Error in mainfile (line 20)
    step4_predict_events(movie_definition, inout_para, prob_para, all_ellipse_info, all_num_ellipses,
    accumulated_jitters);"

Changing prediction type (from global -> density -> etc) does not help.

Do you know what might be causing this?

Thank you!

**Comment by tianchengzhe** [comment]
Can you send me your training data sets to c.z.tian@gmail.com ?

**Comment by tianchengzhe** [comment]
Solved via email.

The problem is that during motion training, it is not allowed to label multiple cells in the same frame with the same Cell ID. For example, after mitosis, only continue labeling one of the daughter cells with the parental Cell ID, not both. The GUI does not check for this problem, but the inference of migration speed will break down.
